README.md

```diff
@@ -64,12 +64,22 @@ Example contents of a `.env` file:
 # Server mode is multi-user and suitable for intranet / internet use
 MODE=
 
+# A comma separated string that defines the available runTimes.
+# Priority is given to the runtime that comes first in the string.
+# Possible options at the moment are sas and js
+
+# options: [sas|js|py|sas,js|js,sas|py,sas|sas,py|py,js|js,py|sas,js,py|sas,py,js|js,sas,py|js,py,sas|py,sas,js|py,js,sas] default:sas
```

Comment:
This list is going to get very long when we add R :-)

Reply:
Yeah, it definitely gonna be very long. So, what should we do? Should we skip these possible permutations of runtimes in readme?

Reply:
Proposal:

```
# This string sets the priority of the available analytic runtimes
# Valid runtimes are SAS (sas), JavaScript (JS) and Python (py)
# For each option provided, there should be a corresponding path, 
# eg SAS_PATH or NODE_PATH
# Priority is given to runtimes earlier in the string
# Example options:  [sas,js,py | js,py | sas | sas,js]
```

Reply:
updated